README.md

```diff
@@ -117,6 +117,60 @@ You can add lists, ranges, and expansion rules as well:
 python3 -m script.intentfest sample_template 'set color to <color> and brightness to {brightness}' --values color red green --range brightness 1 2 --rule color '[the] {color}'
 ```
 
+## Check what the conversation would look like
+
+To quickly simulate a conversation, use:
+
+```sh
+python3 -m script.intentfest conversation --language en # Set your preferred language
+```
+
+For example:
+
+```
+python3 -m script.intentfest conversation --language en
+Write here your query, or type 'a' (or "arrow up" on keyboard), to repeat the last one question. Type in 'r' to reload intents (after changing "sentences" or "responses").
+
+<<< Turn on the lights in the living room
+>>> Turned on lights in living room
+<<< r
+New intents reloaded!
+<<< a
+<<< Turn on the lights in the living room
+>>> Turned on lights in living room sir.
+<<<
+```
+
+You can type **`r`** to reload new responses and / or sentences, and **`a`** (or `arrow up` - something like in terminal using commands history).
+
+There is also an option to see all the response data, just type **`d`** to toggle debug mode, or call the `conversation` module with `--sentence-debug 1` to turn it on immediately:
+
+```
+python3 -m script.intentfest conversation --language en --debug-response 1
```

Comment:
Why not just `--debug-response`? If the parameter included in the args, show debug info. If its' omitted, don't.

Reply:
My suggestion is, regardless of whether you want to add/remove debug functionality at runtime, drop the "1". Just check whether the `--debug-response` parameter is present, not if it has a certain value.

Reply:
Yeah, funny thing 😅 That was my first thought. But I got this error  `__main__.py: error: argument --debug-response: expected one argument`, so that was the reason why I check the value of this parameter 😆. BUT NOW IT IS AS IT SHOULD BE! 😁